app/main.py

```diff
@@ -1,5 +1,71 @@
+from __future__ import annotations
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list = None,
```

Comment:
If argument can be `None` you must show it in type hint. Fix everywhere.
```suggestion
            coords: list | None = None,
```